test/test-applications/integrations/TestApplication.Wcf.Client.NetFramework/TestApplication.Wcf.Client.NetFramework.csproj

```diff
@@ -1,13 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net462</TargetFrameworks>
+    <TargetFramework>net462</TargetFramework>
```

Comment:
I would keep TargetFrameworks, even if you are using only one Framework.
This property is configured at the props file changing it to singular form could leads to unexpected scenarios.